src/app/auth/services/auth.service.ts

```diff
@@ -284,9 +284,12 @@ export const changePassword = async (newPassword: string, currentPassword: strin
 
   // Encrypt the mnemonic
   const encryptedMnemonic = encryptTextWithKey(user.mnemonic, newPassword);
-  const privateKey = Buffer.from(user.privateKey, 'base64').toString();
+  const privateKey = Buffer.from(user.keys.ecc.privateKey, 'base64').toString();
   const privateKeyEncrypted = aes.encrypt(privateKey, newPassword, getAesInitFromEnv());
 
+  const privateKyberKey = Buffer.from(user.keys.kyber.privateKey, 'base64').toString();
```

Comment:
As far as I currently know, if we are going to do this conditionally, we should be careful about Buffering or encrypting keys whose value is not sure to be present